packages/integration_test/lib/src/reporter.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:test_core/src/runner/engine.dart';
+import 'package:test_core/src/runner/reporter.dart';
```

Comment:
This will give warnings right? 

We should probably pin the dependency for this to make sure we don't inadventently get broken by a patch release that changes internals.

I'm also surprised we don't need analysis ignores here, like `// ignore: implementation_import` or something. We should enable that lint and add the ignore here.

Reply:
Yeah the lint isn't set up for this. This is a bit out of scope of this PR though – I have started #3187 to see if we can turn on the lint. There are some existing failures in the repo, will follow up tomorrow.